ui/packages/catalog-ui-search/src/main/webapp/component/filter/filter.view.js

```diff
@@ -45,7 +46,7 @@ module.exports = Marionette.LayoutView.extend({
     this.model.set({
       type: attribute,
       comparator,
-      value: [value],
+      value: [value]
```

Comment:
✏️ I recommend reverting this change. The hanging comma on the last item makes it easier to add new keys in the future and makes it more obvious in the diff